scapy/contrib/bfd.py

```diff
@@ -35,20 +52,88 @@
 }
 
 
+# https://www.rfc-editor.org/rfc/rfc5880 [Page 10]
+_authentification_type = {
+    0: "Reserved",
+    1: "Simple Password",
+    2: "Keyed MD5",
+    3: "Meticulous Keyed MD5",
+    4: "Keyed SHA1",
+    5: "Meticulous Keyed SHA1",
+}
+
+
+class OptionalAuth(Packet):
+    name = "Optional Auth"
+    fields_desc = [
+        ByteEnumField("auth_type", 1, _authentification_type),
+        FieldLenField(
+            "auth_len",
+            None,
+            fmt="B",
+            length_of="auth_key",
+            adjust=lambda pkt, x: int(x + 3) if pkt.auth_type <= 1 else int(x + 8),
```

Comment:
nit: what do you need the `int(...)` for?

Reply:
Don't need thoses int(...) at all '^^